decidim-core/lib/decidim/core/test/shared_examples/simple_event.rb

```diff
@@ -31,7 +31,7 @@
   let(:resource_title) { resource.title["en"] }
   # to be used when resource is a component resource, not a participatory space, in which case should be overriden
   let(:participatory_space) { resource.participatory_space }
-  let(:participatory_space_title) { participatory_space.title["en"] }
+  let(:participatory_space_title) { decidim_escape_translated(decidim_sanitize_translated(participatory_space.title)) }
```

Comment:
This is how it looks like in `develop` 

https://github.com/decidim/decidim/blob/a068bbb39f70d5b56c6ab0c4d4c0618c71d4270f/decidim-core/lib/decidim/core/test/shared_examples/simple_event.rb#L34


```suggestion
  let(:participatory_space_title) { decidim_sanitize_translated(participatory_space.title) }
```

Reply:
Actually, this is handled in #12648 , as in the original PR of ( we used the escape )
![image](https://github.com/decidim/decidim/assets/105683/c0502858-d957-4dd3-b361-0429cd8b11fd)

Reply:
Oh if it's handled in that PR I'll let it pass then 😄